backend/app/assets/javascripts/spree/backend/orders/risky.js

```diff
@@ -0,0 +1,101 @@
+var activeRiskyOrder = 0;
+var totalRiskyOrders;
+
+$(document).ready(loadRiskyOrder);
+
+$(document).on('click', '.js-risky-next', nextRiskyOrder);
+$(document).on('click', '.js-risky-prev', prevRiskyOrder);
+$(document).on('click', '.js-risky-cancel', cancelRiskyOrder);
+$(document).on('click', '.js-risky-approve', approveRiskyOrder);
+$(document).on('click', '.js-view-risky-order', function() {
+  viewRiskyOrder($(this));
+});
+
+function cancelRiskyOrder(){
+  var url = Spree.routes.orders_api + "/" + getOrderNumber() + '/cancel';
```

Comment:
Identifier 'orders_api' is not in camel case.<br>Mixed double and single quotes.<br>'Spree' is not defined.